apps/preview/nuxt/pages/showcases/MegaMenu/MegaMenuNavigation.vue

```diff
@@ -141,16 +141,16 @@
                 </div>
               </SfListItem>
             </li>
-            <template v-for="node in activeMenu.children">
-              <li v-if="node.isLeaf" :key="node.value.label">
+            <template v-for="node in activeMenu.children" :key="node.value.label">
```

Comment:
On the other hand, we'd be happy to merge this improvement. 😄 So it'd be great if you could remove `z-index` stuff, so we could merge just the `key`.

Reply:
No problem @dkacper, reverted [here](https://github.com/vuestorefront/storefront-ui/pull/2779/commits/bbc3e0253d811111692999035015f490c369228b)

Reply:
Thanks